### DISCUSSION TITLE
recommendations for inexpensive gpu? (my board doesn't support HW accel)

### DISCUSSION THREAD

**Discussion by paultanger** [blakeblackshear/frigate#3330] [Ask A Question]
Hi, I have an intel CPU with quicksync, but sadly my [supermicro board](https://www.supermicro.com/en/products/motherboard/X10SLL-F) chipset doesn't support it (C222) so I'm hoping to get a cheap GPU and put into PCI slot to utilize hardware encoding.  Wondering if there is recommendations for compatible GPUs?
It seems AMD/ATI GPUs are more compatible that NVIDIA, like P400 is a good cheap choice ($100), but is there a AMD/ATI similar option that wouldn't require the custom NVIDIA drivers?  I know it says `Radeon HD 2000 and newer GPUs` but curious if anyone had more specific recommendations for < $150?

I also realize I could upgrade the board / CPU / mem but that seems expensive and complicated.
btw, loving Frigate so far, great job!!

If it is relevant, I'm running through TrueNAS SCALE using docker compose.
Thanks!!

**Comment by mdkeil** [comment]
Take a look at AMD Radeon RX 550 Models with UVD 6.3 support.
